code/modules/shuttle/on_move.dm

```diff
@@ -91,6 +91,8 @@ All ShuttleMove procs go here
 			return
 
 	loc = newT
+	if (newT.z != oldT.z && !istype(moving_dock, /obj/docking_port/mobile/supply)) // Cargo shuttles cannot carry living mobs which are, for now, the only things actually interested in Z transitions
```

Comment:
i'd drop the istype, too snowflake imo

Reply:
I'll leave it until I get to talk with ninjanomnom about shuttle movement in general

Reply:
the istype is bad here yeah